2025-08-18 10:06:26 [INFO] tests.auth.test_auth_api: Тест: Пустое тело запроса [test_auth_api.py:134]
2025-08-18 10:06:26 [INFO] core.http_client: Sending POST request to APIEndpoints.AUTH [http_client.py:73]
2025-08-18 10:06:26 [INFO] core.http_client: Received response 403 from http://localhost:4040/APIEndpoints.AUTH [http_client.py:80]
2025-08-18 10:06:26 [INFO] tests.auth.test_auth_api: Тест: Пустое тело запроса [test_auth_api.py:134]
2025-08-18 10:06:26 [INFO] core.http_client: Sending POST request to APIEndpoints.AUTH [http_client.py:73]
2025-08-18 10:06:26 [INFO] core.http_client: Received response 500 from http://localhost:4040/APIEndpoints.AUTH [http_client.py:80]
2025-08-18 10:06:26 [WARNING] utils.allure_utils: Ответ не является валидным JSON, аттачим как текст. [allure_utils.py:62]
2025-08-18 10:06:26 [INFO] tests.auth.test_auth_api: Тест: Пустое тело запроса [test_auth_api.py:134]
2025-08-18 10:06:26 [INFO] core.http_client: Sending POST request to APIEndpoints.AUTH [http_client.py:73]
2025-08-18 10:06:26 [INFO] core.http_client: Received response 403 from http://localhost:4040/APIEndpoints.AUTH [http_client.py:80]
2025-08-18 10:06:26 [INFO] tests.auth.test_auth_api: Тест: Пустое тело запроса [test_auth_api.py:134]
2025-08-18 10:06:26 [INFO] core.http_client: Sending POST request to APIEndpoints.AUTH [http_client.py:73]
2025-08-18 10:06:26 [INFO] core.http_client: Received response 403 from http://localhost:4040/APIEndpoints.AUTH [http_client.py:80]
2025-08-18 10:06:26 [INFO] tests.auth.test_auth_api: Тест: Пустое тело запроса [test_auth_api.py:134]
2025-08-18 10:06:26 [INFO] core.http_client: Sending POST request to APIEndpoints.AUTH [http_client.py:73]
2025-08-18 10:06:26 [INFO] core.http_client: Received response 500 from http://localhost:4040/APIEndpoints.AUTH [http_client.py:80]
2025-08-18 10:06:26 [WARNING] utils.allure_utils: Ответ не является валидным JSON, аттачим как текст. [allure_utils.py:62]
2025-08-18 10:06:26 [INFO] tests.auth.test_auth_api: Тест: Пустое тело запроса [test_auth_api.py:134]
2025-08-18 10:06:26 [INFO] core.http_client: Sending POST request to APIEndpoints.AUTH [http_client.py:73]
2025-08-18 10:06:26 [INFO] core.http_client: Received response 403 from http://localhost:4040/APIEndpoints.AUTH [http_client.py:80]
